src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -653,17 +644,11 @@ protected void incomingMessage(SerialMessage serialMessage) {
 
     @Override
     public void deviceDiscovered(int nodeId) {
-        if (discoveryService == null) {
-            return;
-        }
-        // discoveryService.deviceDiscovered(nodeId);
+        // Nothing to do here
     }
 
     public void deviceAdded(ZWaveNode node) {
```

Comment:
Why not just delete this method?

Reply:
I deleted it and there was a special case that should be handled by the event handler in `ZWaveDiscoveryService` - I added this case